apps/communications/contacts/js/bootstrap.js

```diff
@@ -518,7 +67,10 @@
     utils.PerformanceHelper.domLoaded();
     window.removeEventListener('DOMContentLoaded', ondomloaded);
 
-    applyCache(FIRST_CHUNK);
+    LazyLoader.load('/contacts/js/cache.js', () => {
+      console.log('CACHE loaded, applying');
+      Cache.apply('firstChunk');
+    });
```

Comment:
Could we include the .js directly to the `index.html` and remove the lazy loading? We want to avoid any impact in the performance, so the fastest, the better! :)